### PR TITLE
Memefixes 2

### PR DESCRIPTION
**Pull request by MemeWizard**
#### Changelog:
- Fixed vanilla bug where `GetHerHis` in `Create Treasury` description didn't work, because it had no scope.
- Changed vanilla `MODIFIER_ARMY_REINFORCE_RATE` name, now it's called `Commanded Army Reinforcement Rate` instead of `Levy Reinforcement Rate` because this modifier effect is different from already existing `Levy Reinforcement Rate` modifier.
- Changed vanilla `MODIFIER_CHURCH_OPINION` name, now it's called `Church Opinion` instead of "Temple Vassal Opinion" because this modifier effect is different from already existing `Temple Vassal Opinion` modifier.
- Fixed that Doomhammer is crafted in 583 or 603 instead of unknown times.
- Fixed option localization in `Worgens in Province`, `WCDRU.5005` event.
- Fixed triggers in `The Scarlet Crusade and the Argent Dawn`. Now, this event can fire if inhuman beast holds the `Stratholme` and `Lordaeron`, even if their religion is **Holy Light**.
- Fixed that fourth wall is broken by default, now patron trait is only gained if you enable it in the game rules.

#### Developer changelog:
- Merged `wc_2_DE_customizable_localisation_artifact_names.txt` and `wc_customisable_localisation_DE.txt`.
- Changed Localization of 1 suffix and 1 prefix in artifact localization.
- Hide scope changes in a few events.
- Fixed bug where AI Hakkari characters couldn't sacrifice prisoners.
- Fixed that `Goldrinn` characters not immune to worgen curse and `Goldrinn` provinces not immune to the Plague.

### CODE REVIEW

File: localisation/00_wc_random_artifact_names.csv [hunk 1]
```diff
@@ -177,7 +177,7 @@ artifact_jewelry_rare_prefix_intrigue_2;Disgraced's;;Des Entehrten;;;;;;;;;;;x
 artifact_jewelry_rare_prefix_intrigue_3;Enveloped;;Umh�llter;;;;;;;;;;;x
 artifact_jewelry_rare_prefix_intrigue_4;Snake's;;Finsterer;;;;;;;;;;;x
 artifact_jewelry_rare_prefix_intrigue_5;Spider's;;Der Spinne;;;;;;;;;;;x
-artifact_jewelry_rare_prefix_intrigue_6;Malevolence;;B�swilliger;;;;;;;;;;;x
+artifact_jewelry_rare_prefix_intrigue_6;Malevolent;;B�swilliger;;;;;;;;;;;x
```
Comment (MemeWizard) [inline localisation/00_wc_random_artifact_names.csv]: @arithon, check localization changes, pls

Comment (arithon) [reply localisation/00_wc_random_artifact_names.csv]: only these two lines? 180 and 400? looks good, I think

Comment (MemeWizard) [reply localisation/00_wc_random_artifact_names.csv]: Yeah, only these two lines. Thanks <3

**Comment by MemeWizard** [comment]
I thought you will want to change german loc too, but ok

**Pull request merged**